.github/workflows/datadog-static-analysis.yml

```diff
@@ -0,0 +1,21 @@
+on: [push]
+
+name: Datadog Static Analysis
+
+jobs:
+  static-analysis:
+    runs-on: ubuntu-latest
+    name: Datadog Static Analyzer
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Check code meets quality and security standards
+      id: datadog-static-analysis
+      uses: DataDog/datadog-static-analyzer-github-action@v1
+      with:
+        dd_api_key: ${{ secrets.SYSTEM_TESTS_E2E_DD_API_KEY }}
+        dd_app_key: ${{ secrets.SYSTEM_TESTS_E2E_DD_APP_KEY }}
```

Comment:
@juli1 Why are we using these keys? There are other available and these belong to system tests, AFAIK.

Reply:
It seems there are only API keys. I do not have access to the settings - is it possible to set up secrets specific to the static analyzer (like `STATIC_ANALYZER_APP_KEY` and `STATIC_ANALYZER_APO_KEY`)?

Reply:
FYI this is the approach we [followed on the agent](https://github.com/DataDog/datadog-agent/pull/26627/files)

Reply:
@juli1 We can set up specific secrets. I think it's better this way.

Reply:
Thank you. Can you tell me when an app and api keys are set and what is the variable name?

Thank you.